### ISSUE TITLE
NullPointerException when trying to add a product with no image

### ISSUE THREAD

**Issue by david-r-edgar** [projectwife/mtesitoo-android#92]
Running on the emulator, `pt-bugfixes` branch.

I click 'Add' to add a new product.
I fill in all the fields with arbitrary values. But I don't add a new image.
I expect the product to be added with no images.
But instead a crash is reported - but then Tesitoo keeps running.
The product appears to be added, however.

If instead, I add an image (either from the camera or from the gallery), then the product is added as expected, with no crash.

logcat shows a NullPointerException:

```
04-26 20:32:31.212 3591-3591/com.tesitoo D/Product: Product{id+'0'name='milk', description='milky', price='238', thumbnail=null
04-26 20:32:31.212 3591-3591/com.tesitoo D/Submit Product URL: https://tesitoo.com/api/v1//product/product
04-26 20:32:31.798 3591-3591/com.tesitoo D/AndroidRuntime: Shutting down VM
04-26 20:32:31.798 3591-3591/com.tesitoo E/AndroidRuntime: FATAL EXCEPTION: main
                                                           Process: com.tesitoo, PID: 3591
                                                           java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.net.Uri.getScheme()' on a null object reference
                                                               at android.content.ContentResolver.openInputStream(ContentResolver.java:630)
                                                               at com.mtesitoo.backend.service.ProductRequest.submitProductThumbnail(ProductRequest.java:123)
                                                               at com.mtesitoo.AddProductActivity2$6.onResult(AddProductActivity2.java:359)
                                                               at com.mtesitoo.AddProductActivity2$6.onResult(AddProductActivity2.java:355)
                                                               at com.mtesitoo.backend.service.ProductThumbnailResponse.onResponse(ProductThumbnailResponse.java:45)
                                                               at com.android.volley.toolbox.StringRequest.deliverResponse(StringRequest.java:60)
                                                               at com.android.volley.toolbox.StringRequest.deliverResponse(StringRequest.java:30)
                                                               at com.android.volley.ExecutorDelivery$ResponseDeliveryRunnable.run(ExecutorDelivery.java:99)
                                                               at android.os.Handler.handleCallback(Handler.java:739)
                                                               at android.os.Handler.dispatchMessage(Handler.java:95)
                                                               at android.os.Looper.loop(Looper.java:135)
                                                               at android.app.ActivityThread.main(ActivityThread.java:5254)
                                                               at java.lang.reflect.Method.invoke(Native Method)
                                                               at java.lang.reflect.Method.invoke(Method.java:372)
                                                               at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
                                                               at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
```

**Comment by ptyagicodecamp** [comment]
Fixed in https://github.com/projectwife/mtesitoo-android/pull/105

**Comment by david-r-edgar** [comment]
verified fixed in 7a9c97a27271e7f65496aec316eaee380161e4c5
